README.md

```diff
@@ -303,12 +305,27 @@ trigram-index = true
 # in memory. This determines how often it will check FS
 # for new or deleted metrics.
 scan-frequency = "5m0s"
+# Control trie index
+#  This index is built as an alternative to trigram index, with shorter indexing
+#  time and less memory usage (around 2 - 5 times). For most of the queries, trie
+#  is faster than trigram. For queries with keyword wrap around by widcards (like
+#  ns1.ns2.*keywork*.metric), trigram indexi performs better. Trie index could
+#  be speeded up by enabling adding trigrams to trie, at the cost of memory usage (
+#  depends on the facotr, the memory usage of trie would be at least doubled).
+trie-index = false
+trie-index-with-trigrams = false
```

Comment:
Would it be possible to use 
trigram-index = true && trie-index = true 
in order to specify
trie-index-with-trigrams = true
? 
3 parameters in the config creates a complexity of understanding which combination of them is acceptable and which not.

Reply:
It's not a bad idea. Let me think about it and see if it's possible/good.